lib/pf/config.pm

```diff
@@ -830,6 +830,23 @@ sub is_omapi_enabled {
     return $FALSE;
 }
 
+=item is_omapi_configured
+
+Check if required OMAPI configuration parameters (omapi.key_name & omapi.key_base64) are present before configuring it
+
+=cut
+
+sub is_omapi_configured {
+    return $FALSE unless $Config{'omapi'}{'host'} eq "localhost";
+
+    if ( ($Config{'omapi'}{'key_name'} && $Config{'omapi'}{'key_name'} ne '') && ($Config{'omapi'}{'key_base64'} && $Config{'omapi'}{'key_base64'} ne '') ) {
+        return $TRUE;
+    }
+
+    $logger->warn("OMAPI lookup is locally enabled but missing required configuration parameters 'key_name' and/or 'key_base64'");
```

Comment:
Yep but it is a pretty generic message... It is covering up if someone messes with the default one... and since it is the values we are checking I output them